torch/csrc/api/src/nn/modules/linear.cpp

```diff
@@ -38,5 +38,36 @@ Tensor LinearImpl::forward(const Tensor& input) {
   AT_ASSERT(!options.with_bias_ || bias.defined());
   return torch::linear(input, weight, bias);
 }
+
+BilinearOptions::BilinearOptions(int64_t in1, int64_t in2, int64_t out) : in1_(in1), in2_(in2), out_(out) {}
+
+BilinearImpl::BilinearImpl(BilinearOptions options) : options(options) {
+  reset();
+}
+
+void BilinearImpl::reset() {
+  weight =
+      register_parameter("weight", torch::empty({options.out_, options.in1_, options.in2_}));
+  if (options.with_bias_) {
+    bias = register_parameter("bias", torch::empty(options.out_));
+  }
```

Comment:
We should register an undefined tensor (i.e. `Tensor()`) when `options.bias()` is false, to better match the Python version.